doc/classes/RigidBody2D.xml

```diff
@@ -214,7 +214,7 @@
 		</signal>
 		<signal name="sleeping_state_changed">
 			<description>
-				Emitted when [member sleeping] changes.
+				Emitted when the physics engine changes the body's [member sleeping] state.
```

Comment:
I'd remove the reference to `[member sleeping]` here (just use the normal "sleeping" word), and maybe replace it with an extra clarification, e.g.:
```
[b]Note:[/b] Changing the value `[member sleeping]` will not trigger this signal by itself. It is only emitted if the sleeping state then changes in the physics engine as a result of the property change.
```

Reply:
Updated.
Note: I've changed the wording from your suggestion slightly and add the use of `emit_signal("sleeping_state_changed")` to trigger it manually.